tests/operators/functionals/test_l1_norm.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for L1-functional."""
+
+# Copyright 20234 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import pytest
+import torch
+from mrpro.operators.functionals.l1 import L1Norm
+
+
+@pytest.mark.parametrize(
+    (
+        'x',
```

Comment:
suggestion:
forward_x, prox, prox_complex_conjugate

and then do something like
torch.testing.assert_close(l1_norm.forward(x), forward_x, rtol=1e-3, atol=1e-3)

why are p_forward and pcc_forward needed? the forward of the norm is already tested with forward_x, isn't it?